src/backend/db/products.js

```diff
@@ -11,53 +11,62 @@ export const products = [
     name: "Jordan Zoom",
     price: "100$",
```

Comment:
you can Include a numerical value for price instead of a string, because later you may need to compare it in order to implement sort filter.